src/commonMain/kotlin/org/kbson/BsonDateTime.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kbson
+
+import org.kbson.ext.getCurrentTimeInMillis
+
+/** A representation of the BSON DateTime type. */
+class BsonDateTime(val value: Long) : BsonValue(), Comparable<BsonDateTime> {
```

Comment:
Missing documentation for what this value actually is? I assume it is millis since epoch.

Reply:
Done